shared/constants/chat2/meta.js

```diff
@@ -0,0 +1,325 @@
+// Meta manages the metadata about a conversation. Participants, isMuted, reset people, etc. Things that drive the inbox
+// @flow
+import * as I from 'immutable'
+import * as RPCChatTypes from '../types/rpc-chat-gen'
+import * as RPCTypes from '../types/rpc-gen'
+import * as Types from '../types/chat2'
+import type {_ConversationMeta} from '../types/chat2/meta'
+import {formatTimeForConversationList} from '../../util/timestamp'
+import {globalColors} from '../../styles'
+import {isMobile, isIOS, isAndroid} from '../platform'
+import {parseFolderNameToUsers} from '../../util/kbfs'
+import {toByteArray} from 'base64-js'
+
+const IGNORE_UNTRUSTED_CACHE = false
+IGNORE_UNTRUSTED_CACHE && console.log('aaa NOJIMA skipping locametadata')
+
+const conversationMemberStatusToMembershipType = (m: RPCChatTypes.ConversationMemberStatus) => {
```

Comment:
What is the point of this>

Reply:
i want more UI specific enums passing around and not plumbing through the core ones. also makes this more readable so we can have strings internally so debugging is easier and logging is easier